edx-olive/edx-platform
# [SE-3253] Update new translations for CCX email

This PR updates the translation text for the CCX email.

**Testing Instuctions:** 

1. Login to Studio and look for a CCX course - like [this one](https://studio.stage.campus.gov.il/course/course-v1:OpenCraft+CS101+2020_T1). Ensure that `Enable CCX` is set to True in Advanced Settings.
2. Go to the course page on LMS (or click on "view live").
3. Navigate to Instructor dashboard -> Membership tab.
4. In the "Course Team Management" section, select "CCX Coaches" in the "Select a course team role:" dropdown.
5. Enter your username or email address (should have staff permission) and click on "Add CCX coach" button.
6. Check your inbox for the email.

**Reviewers:**
